lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -94,3 +95,83 @@ def test_no_certificate(self):
             resp.data['error_code'],
             'no_certificate_for_user',
         )
+
+
+@ddt.ddt
+class CertificatesListRestApiTest(AuthAndScopesTestMixin, SharedModuleStoreTestCase, APITestCase):
```

Comment:
Can you add a test for SQL query count so we can verify that it doesn't grow linearly?

Reply:
Sure.